Dikke Ploaten/View/Extensions.swift

```diff
@@ -27,3 +27,24 @@ extension UIViewController {
 	}
 	
 }
+
+extension UITableView {
+	
+	func setEmptyMessage(_ message: String) {
+		let messageLabel = UILabel(frame: CGRect(x: 0, y: 0, width: self.bounds.size.width, height: self.bounds.size.height))
+		messageLabel.text = message
+		messageLabel.textColor = .black
+		messageLabel.numberOfLines = 0;
+		messageLabel.textAlignment = .center;
+		messageLabel.font = UIFont(name: "TrebuchetMS", size: 15)
+		messageLabel.sizeToFit()
+		
+		self.backgroundView = messageLabel;
+		self.separatorStyle = .none;
+	}
+	
+	func restore() {
```

Comment:
Zou ik niet in een andere methode zetten, maar de `message` in `setEmptyMessage` nullable maken, als die `nil` is, dan leeg je de `backgroundView`, anders stel je die in.